### DISCUSSION TITLE
Saw this video over the weekend: LangGraph Studio: The first agent IDE - quite interesting

### DISCUSSION THREAD

**Discussion by cyfyifanchen** [TEN-framework/ten-framework#158] [Ideas]
https://www.youtube.com/watch?v=pLPJoFvq4_M&t=243s

LangChain went to a different approach VS. Dify and Coze. Wondering what's the next.

**Comment by cyfyifanchen** [comment]
I am pretty sure once our graphic designer is released, we are gonna have similar videos.

**Reply by adlermore** [reply]
It sounds like you're planning to create similar videos once your graphic designer is available, which could be a great way to enhance your content! If you need any support with planning or technical aspects for those videos, feel free to reach out.
